apps/src/templates/instructions/TopInstructions.jsx

```diff
@@ -667,7 +667,7 @@ class TopInstructions extends Component {
       mapReference || (referenceLinks && referenceLinks.length > 0);
 
     const displayHelpTab =
-      (levelVideos && levelVideos.length > 0) || levelResourcesAvailable;
+      (levelVideos && levelVideos.length > 0) || !!levelResourcesAvailable;
```

Comment:
this changes `displayHelpTab` to be a boolean value.  The only place it is used is as a prop for `TopInstructionsHeader` which is expecting a boolean.